packages/core/src/item.ts

```diff
@@ -52,7 +52,7 @@ export interface FieldDef {
 export const FIELD_DEFS: { [t in FieldType]: FieldDef } = {
     [FieldType.Username]: {
         type: FieldType.Username,
-        pattern: ".*",
+        pattern: /.*/,
```

Comment:
Not a bad idea switching to a regex literal, but please make sure it doesn't interfere with this: https://github.com/padloc/padloc/blob/v4/packages/app/src/elements/field.ts#L341